namer/consul/src/main/scala/io/buoyant/namer/consul/ConsulInitializer.scala

```diff
@@ -68,9 +68,9 @@ case class ConsulConfig(
     }
 
     val service = Http.client
-      .withMonitor(interruptionMonitor)
       .withParams(Http.client.params ++ params)
-      .configured(Label("namer" + prefix))
+      .withLabel(prefix.show)
```

Comment:
(1) withParams comes first so that withMonitor overrides any default monitors in the params.

(2) this should probably be `prefix.show.stripPrefix("/")`